fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -483,7 +483,7 @@ public void readFields(DataInput in) throws IOException {
         if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_81) {
             replicaQuotaSize = in.readLong();
         } else {
-            replicaQuotaSize = FeConstants.default_db_replica_quota_size;
+            replicaQuotaSize = Config.default_db_replica_quota_bytes;
```

Comment:
If change this to Config, different FE may has different configuration. So this will result in inconsistency of meta data.

This is just a default value filled in for compatibility with old data. I suggest to leave it as `FeConstants.default_db_replica_quota_size`

Reply:
Hi morningman:
You are right, I restore the modification of default_db_replica_quota_size.